mqtt/mqtt-policy/Cargo.toml

```diff
@@ -17,4 +17,5 @@ policy = { path = "../policy" }
 [dev-dependencies]
 assert_matches = "1.3"
 bytes = "0.5"
+proptest = "0.9"
```

Comment:
I guess you can remove this line, because you already have the optional include back?

Reply:
those are different Cargo.toml files. one from broker, another from policy.